orbit/radius.py

```diff
@@ -451,6 +451,17 @@ def oops_button_hover(self):
     def get_automated_feedback(self, attr):
         if (gbl := getattr(self, attr)) is None:
             return 'No submission'
+
+        if attr == 'init':
+            due_date = int(self.assignment.initial_due_date)
+        elif attr == 'final':
+            due_date = int(self.assignment.final_due_date)
+        else:
+            due_date = 0
```

Comment:
`match attr:`?

Reply:
sure but there are only two here and this is fewer lines